### DISCUSSION TITLE
AP Mode

### DISCUSSION THREAD

**Discussion by MTRobin** [espressif/esp-iot-solution#407] [Ideas]
This setup will work for my project, but I would like it to run as an access point, instead of a station. I am trying to connect the ESP32-S3 to my computer (PC, Mac, or Linux) and have it see the ESP32 as a network card. Then the ESP32 would work as an access point, and any device that connects to it with then be on the same network as the computer and can communicate.

Bonus features would be to a DHCP server that would set IP addresses on both the computer and the wireless devices.

Thank you,
Maxwell

**Comment by leeebo** [comment]
Please try with esp-iot-bridge project https://github.com/espressif/esp-iot-bridge/issues/86
